README.md

```diff
@@ -156,7 +171,7 @@ Their engagement with *The Turing Way* project and community should help them un
 
 ## References
 
-- _Sharan, Malvika, & Mehonic, Aida. (2022, July 21). Community Managers and Research Application Managers at the Tools, Practices and Systems Programme at The Alan Turing Institute. Zenodo. https://doi.org/10.5281/zenodo.6878124_
```

Comment:
Is this reference not going to be added?

Reply:
This is an outdated reference - I think we should upload this in the CM resources (it is already in our shared drive) but not any longer suitable as a main source.